BUILD

```diff
@@ -1,23 +0,0 @@
-java_binary(
-    name = "json_test",
```

Comment:
By convention, we use `all-small-and-hyphens-between-words` for target names

Reply:
But Java does not allow hyphens in variable names, so if we want to call this class from other files, we wont be able to?

Reply:
It's the target which will be named with hyphens. It's independent of the class name